Jenkinsfile

```diff
@@ -1,9 +1,9 @@
 #!/usr/bin/env groovy
 
 node('rhel7'){
-	tools {
-		jdk 'openjdk-11'
-	}
+	def javaHome = tool 'openjdk-11'
+	env.JAVA_HOME = "${mvnHome}/bin"
```

Comment:
Don't know why the the tools block does not work but it seems to me that `${mvnHome}` shoud be replaced by `${javaHome}`